docs/tips/19-dangerously-set-inner-html.md

```diff
@@ -6,7 +6,7 @@ permalink: tips/dangerously-set-inner-html.html
 prev: children-undefined.html
 ---
 
-Improper use of the `innerHTML` can open you up to a [cross-site scripting (XSS)](https://en.wikipedia.org/wiki/Cross-site_scripting) attack.  Sanitizing user input for display is notoriously error-prone, and failure to properly sanitize is one of the [leading causes of web vulnerabilities](https://owasptop10.googlecode.com/files/OWASP%20Top%2010%20-%202013.pdf) on the internet.
+Improper use of the `innerHTML` can open you up to a [cross-site scripting (XSS)](https://en.wikipedia.org/wiki/Cross-site_scripting) attack.  Sanitizing user input for display is notoriously error-prone, and failure to properly sanitize is one of the [leading causes of web vulnerabilities](https://owasptop10.googlecode.com/files/OWASP%20Top%2010%20-%202013.pdf) on the Internet.
```

Comment:
[Lowercase "internet" is fine now](http://www.nytimes.com/2016/05/25/business/media/internet-to-be-lowercase-in-new-york-times-and-associated-press.html), let's leave that.